disco/preprocess/select_timepoints2.py

```diff
@@ -495,6 +495,7 @@ def main(
         compression_rate: ratio between the number of critical timepoints and the total number of timepoints in the timeseries
 
     """
+    critical_conditions = set(critical_conditions)
```

Comment:
Is it okay to place it here? I couldnt figure out where this function is called. You can recommend another location if that is better - before it is passed to main.

Reply:
This works.